MAUI/Shimmer/Customization.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Customization of Syncfusion .NET MAUI Shimmer (SfShimmer) control, its basic features, elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Customization of .NET MAUI Shimmer (SfShimmer)
+
+The `SfShimmer` control provides the following properties to customize the Shimmer,
+
+* `Fill` - To customize the background color of Shimmer view. 
+* `WaveColor` - To customize the Shimmer wave color.
+* `WaveWidth` - To customize the Shimmer wave width.
+* `WaveDirection` - To customize the Shimmer wave direction.
+* `RepeatCount` - To customize the repeat count for the Shimmer.
+* `AnimationDuration` - To customize the animation duration of the wave.
+
+This section explains how to customize the Shimmer control.
+
+## Fill
+
+The background color of the `SfShimmer` view can be customized by using the `Fill` property. The default value of `Fill` property is "#F3EDF7" Color.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" Fill="Pink">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      Fill = Colors.Pink,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveColor
+
+The color of wave in `SfShimmer` view can be customized by using the `WaveColor` property. The default value of `WaveColor` is `#F3EDF7` Color.
```

Comment:
The wave color in the `SfShimmer` view can be customized using the `WaveColor` property. The default value of `WaveColor` is the `#F3EDF7` Color.

Reply:
Content modified